tests/integration/mixer/policy/ratelimiting_test.go

```diff
@@ -493,3 +647,78 @@ spec:
   - name: istio-ingressgateway
     namespace: istio-system
 `
+var egressConfig = `
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: cnn
+spec:
+  hosts:
+  - edition.cnn.com
+  ports:
+  - number: 80
+    name: http-port
+    protocol: HTTP
+  - number: 443
+    name: https
+    protocol: HTTPS
+  resolution: DNS
+  location: MESH_EXTERNAL
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-egressgateway
+spec:
+  selector:
+    istio: egressgateway
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - edition.cnn.com
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: egressgateway-for-cnn
+spec:
+  host: istio-egressgateway.istio-system.svc.cluster.local
+  subsets:
+  - name: cnn
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: direct-cnn-through-egress-gateway
+spec:
+  hosts:
+  - edition.cnn.com
+  gateways:
+  - <bookinfo_namespace>/istio-egressgateway
```

Comment:
is gateway really in a bookinfo namespace? i thought it's in istio-system?

Reply:
It is.. we apply egressconfig in bookinfo ns:
g.ApplyConfigOrFail(t,
			bookinfoNs,
			strings.Replace(egressConfig, "<bookinfo_namespace>", bookInfoNameSpaceStr, -1))

Reply:
@kyessenov Please note that a user can deploy any number of ingress/egress gateways, in any namespace.